kueue-ks/controllers/clustermetrics_controller.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2024 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	clustermetrics "kubestellar/galaxy/clustermetrics/api/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	//"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	v1beta1 "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+)
+
+// ClusterMetricsReconciler reconciles a ClusterMetrics object
```

Comment:
Thanks @cwiklik for submitting this PR for kueue - overall it looks good. Could you pls. add in the comment what is the purpose of this clustermetrics_controller (to distinguish this from the cluster metrics controller that already exists ) - I assume it's about updating quotas automatically based on cluster metrics but it would be good to explain.